backend/src/zac/contrib/kownsl/views.py

```diff
@@ -113,7 +114,18 @@ def get(self, request, request_uuid):
             if request.user.username in review_users
             else "false"
         }
-        return Response(review_request, headers=headers)
+        zaak_url = review_request["for_zaak"]
+        zaak = get_zaak(zaak_url)
+        return_url = get_ui_url(
+            [
+                "ui",
+                "zaken",
+                zaak.bronorganisatie,
+                zaak.identificatie,
+            ]
+        )
+        data = {**review_request, "return_url": return_url}
```

Comment:
this is called `frontend_url` in the Kownsl schema, not `return_url`

```suggestion
        data["frontend_url"] = return_url
```

Reply:
Are you sure?

From https://kownsl.utrechtproeftuin.nl/api/docs/:
frontend_url: The frontend URL for reviewers to submit their review.

Looking at https://github.com/GemeenteUtrecht/kownsl/blob/master/src/kownsl/api/serializers.py this seems to refer back to "core:detail" of kownsl. 

This is to go return to the zaak-detail page.

Reply:
in that case I am wrong - but I don't understand why the backend then has to provide that URL?